File: Data Structures & Algorithms/max-consecutive-ones/submission-1.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int sequence, maxSequence = 0;
        for (int i = 0; i < nums.length; i++) {
            sequence = 0;
            for (int j = i; j < nums.length && nums[j] == 1 ; j++, i++, sequence++);
            maxSequence = sequence > maxSequence ? sequence : maxSequence;
        }

        return maxSequence;
    }
}

File: Data Structures & Algorithms/duplicate-integer/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        /* 
            Criar um Hashset
            Percorrer os itens verificando se estão no hashset,
             - Se achar encontra retorna true
             - Se não adiciona valor no hashset
        */
        const hashset = new Set();

        for(const num of nums) {
            if (hashset.has(num)) {
                return true;
            }
            hashset.add(num);
        }

        return false
    }
}
